tests/e2e/test-site/autoscript.js

```diff
@@ -0,0 +1,91 @@
+(function () {
+  const rxFlatMap = rxjs.operators.flatMap;
+  const rxMap = rxjs.operators.map;
+  const rxTap = rxjs.operators.tap;
+  const url = new URL(window.location.href);
+  const clientId = url.searchParams.get('clientId');
+  const scope = url.searchParams.get('scope');
+  const tenant = url.searchParams.get('tenant');
+  const tree = url.searchParams.get('UsernamePassword');
```

Comment:
`tree` can't be configurable because the test is specifically looking for `NameCallback` and `PasswordCallback`, which are only reliably available in the "UsernamePassword" tree.  So `tree` should be set to that known value.

`const tree = 'UsernamePassword';`